### DISCUSSION TITLE
Route network connections of one container through another container

### DISCUSSION THREAD

**Discussion by askoa** [containers/podman#23991] [Q&A]
I am using podman without root privileges. I want to install a proxy like mitm or sock5 or install vpn like wireguard on one container and setup a second container such that all its internet connections go through the first container.

Basically I want to know how to use one container as network gateway for another container in rootless mode. I looked up online and did not find any answers.

**Comment by Luap99** [comment]
you can use `--network container:<vpn ctr name>` to share the same netns from the vpn container with another container

**Reply by connelhooley** [reply]
Is sharing the same network namespace the same thing as routing traffic through a container?
